src/worker/generate-zip.ts

```diff
@@ -0,0 +1,5 @@
+import JSZip = require("jszip");
```

Comment:
My bad: this is the way that the compiler actually wants things to be compiled 😩:


```suggestion
import * as JSZip from"jszip";
```

This will solve the failing build.